docs/src/components/mdx/index.tsx

```diff
@@ -3,13 +3,18 @@ import DSLink from '@pluralsight/ps-design-system-link'
 import * as Text from '@pluralsight/ps-design-system-text'
 import cx from 'classnames'
 import { Link as GatsbyLink } from 'gatsby'
-import React, { HTMLAttributes } from 'react'
+import React, { AllHTMLAttributes, HTMLAttributes } from 'react'
```

Comment:
what's the diff between AllHTMLAttributes and HTMLAttributes

Reply:
One screams about not having `href`, and the other one's ok.